src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/RequestHandler.java

```diff
@@ -24,13 +22,16 @@
 import java.util.*;
 import java.util.stream.Collectors;
 
+import static com.minecolonies.api.util.constant.Suppression.RAWTYPES;
+import static com.minecolonies.api.util.constant.Suppression.UNCHECKED;
+
 /**
  * Class used to handle the inner workings of the request system with regards to requests.
  */
 public final class RequestHandler
 {
 
-    @SuppressWarnings(Suppression.UNCHECKED)
+    @SuppressWarnings(UNCHECKED)
     public static <Request extends IRequestable> IRequest<Request> createRequest(final IStandardRequestManager manager, final IRequester requester, final Request request)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00119)